tsconfig.base.json

```diff
@@ -7,7 +7,9 @@
     "sourceMap": true,
     "target": "es2019" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
     "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
-    "strict": false /* Enable all strict type-checking options. */,
+    "strict": true /* Enable all strict type-checking options. */,
+    "strictPropertyInitialization": false,
+    "strictNullChecks": false,
```

Comment:
remove properties. especially `strictNullChecks: false` opens up for potential runtime crashes